overlord/fdestate/fdemgr_test.go

```diff
@@ -59,36 +91,40 @@ func (u *instrumentedUnlocker) Relock() {
 	u.relocked += 1
 }
 
-func (s *fdeMgrSuite) TestGetManagerFromState(c *C) {
-	st := state.New(nil)
-
+func (s *fdeMgrSuite) startedManager(c *C) *fdestate.FDEManager {
 	defer fdestate.MockDMCryptUUIDFromMountPoint(func(mountpoint string) (string, error) {
 		switch mountpoint {
 		case dirs.SnapdStateDir(dirs.GlobalRootDir):
 			return "aaa", nil
 		case dirs.SnapSaveDir:
 			return "bbb", nil
 		}
-		return "", fmt.Errorf("unknown mount point")
+		panic(fmt.Sprintf("missing mocked mount point %q", mountpoint))
 	})()
 
 	defer fdestate.MockGetPrimaryKeyHMAC(func(devicePath string, alg crypto.Hash) ([]byte, []byte, error) {
-		c.Check(devicePath, Equals, "/dev/disk/by-uuid/aaa")
+		c.Assert(devicePath, Equals, "/dev/disk/by-uuid/aaa")
 		c.Check(alg, Equals, crypto.Hash(crypto.SHA256))
 		return []byte{1, 2, 3, 4}, []byte{5, 6, 7, 8}, nil
 	})()
 
 	defer fdestate.MockVerifyPrimaryKeyHMAC(func(devicePath string, alg crypto.Hash, salt, digest []byte) (bool, error) {
-		c.Check(devicePath, Equals, "/dev/disk/by-uuid/bbb")
+		c.Assert(devicePath, Equals, "/dev/disk/by-uuid/bbb")
 		c.Check(alg, Equals, crypto.Hash(crypto.SHA256))
 		c.Check(salt, DeepEquals, []byte{1, 2, 3, 4})
 		c.Check(digest, DeepEquals, []byte{5, 6, 7, 8})
 		return true, nil
```

Comment:
will we have a variant where this fails?

Reply:
yes, there isn't one now, but it's TBD